.github/workflows/bench_trends.yml

```diff
@@ -12,6 +12,10 @@ on:
   # pull_request:
   #   branches: [ main ]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
```

Comment:
This only runs on main, we probably dont' need it here

Reply:
I added it because I saw that this workflow can run if in PR (only when testing) and because I imagine there's a situation where 2 PRs land at similar times and we only care about the last one.